packages/cli/cli/src/cliV2.ts

```diff
@@ -79,6 +79,16 @@ export function addGeneratorCommands(cli: Argv<GlobalCliOptions>, cliContext: Cl
                             // which this feels better than.
                             description:
                                 "The APIs to list the generators for. If not specified, the generator will be upgraded for all APIs."
+                        })
+                        .option("included-modes", {
```

Comment:
Let's use `include-mode` (singular) so that it'll be natural to migrate to `--include` someday (if ever). Same thing with `exclude-mode` below.